scripts/testing_scripts/test_all_sandia

```diff
@@ -768,8 +804,8 @@ setup_env() {
   done
 
   if [ -e ${CM_ALL_SCRIPT_PATH}/update_lib.sh ]; then
-     echo "calling ${CM_ALL_SCRIPT_PATH}/update_lib.sh $MACHINE"
-     source ${CM_ALL_SCRIPT_PATH}/update_lib.sh $MACHINE
+     echo "calling ${CM_ALL_SCRIPT_PATH}/update_lib.sh $MACHINE $compiler"
+     source ${CM_ALL_SCRIPT_PATH}/update_lib.sh $MACHINE $compiler
```

Comment:
Where is the second argument used?

Reply:
I just pushed changes to the update_lib.sh using this argument, missed those changes in initial PR, thanks for the catch